app/assets/stylesheets/style.css

```diff
@@ -0,0 +1,272 @@
+@import url('https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css');
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:opsz,wght@6..12,300;6..12,400;6..12,700&display=swap');
+
+body {
+  font-family: 'Proxima Nova Light', 'Nunito Sans', sans-serif;
+  font-size: 15px;
+  font-weight: 300;
+  color: #aaa;
+  background-color: #f0f2f1;
+}
+
+a {
+  text-decoration: none;
+}
+
+img {
+  max-width: 100%;
+}
+
+::placeholder {
+  color: #ccc;
+}
+
+.notice {
+  color: green;
+}
+
+.alert {
+  color: red;
+}
+
+.header-wrapper {
+  background-color: #3778c2;
+}
+
+.header-wrapper a {
+  color: #fff;
+}
+
+.bg-body-tertiary {
+  --bs-bg-opacity: 0;
+}
+
+.navbar-action {
+  width: 1.5em;
+}
+
+.navbar-toggler {
+  border: none;
+  padding: 4px 0;
+}
+
+.navbar-toggler-icon {
+  background-image: url("user-svg.svg");
+}
+
+.navbar-collapse {
+  flex-grow: 0;
+}
+
+.navbar-nav {
+  text-align: right;
+}
+
+.navbar-brand {
+  flex-grow: 1;
+}
+
+.header-row {
+  padding: 20px;
+}
+
+.header-row h1,
+.navbar-brand {
+  font-family: 'Proxima Nova Bold', 'Nunito Sans', sans-serif;
+  font-size: 18px;
+  font-weight: 300;
+  text-transform: uppercase;
+  color: #fff;
+  text-align: center;
+}
+
+.main-wrapper {
+  padding: 0;
+}
+
+.content-button a {
+  text-align: center;
+}
+
+.devise-links a {
+  color: #aaa;
+}
+
+.payments-container-edit a {
+  color: #aaa;
+}
+
+.group-item-name a {
+  margin-left: 25px;
+  color: #aaa;
+}
+
+.group-item-action a {
+  color: #aaa;
+}
+
+.home-container {
+  margin: 0;
+  padding: 0 0 50px 0;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  box-shadow: 0 2px 2px 2px #ccc;
+  height: 90vh;
+}
+
+.home-logo {
+  padding: 50px 20px;
+}
+
+.home-link a,
+.content-button a,
+.content-button button {
+  display: block;
+  margin: 5px auto;
+  padding: 20px;
+  width: 100%;
+  max-width: 373px;
+  background-color: #5fb523;
+  color: #fff;
+  font-size: 18px;
+  text-transform: uppercase;
+  text-decoration: none;
+  border: none;
+}
+
+.home-link a:hover,
+.content-button a:hover {
+  background-color: #8ad457;
+}
+
+.content-button button {
+  margin-top: 50px;
+}
```

Comment:
Your app is working well, and with awesome UI 👌 However, it will be a good idea if we can add a few tweaks to the design aspect so as to improve the look and feel. Please take note that it's one of the project's objectives that our design closely mimics the given [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). Let's improve upon the design aspect by introducing the following tweaks:

- [ ] Let's ensure the `Add Category`, `Save Category` and all similar green button is always sticky to the bottom of the screen, and has an inner rectangular white border like can be seen in the template design.

**Your design:**
**Your splash screen:**        |  **Your categories page:**        |  **Your transaction page**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/balatstar/budgetbuddy/assets/12745474/7d3a8409-8eec-4d14-b510-507b372c8d68"> | <img width="800" alt="Home Page" src="https://github.com/balatstar/budgetbuddy/assets/12745474/1c41e5d3-8821-4ccf-86ec-c261bd1f0ea9"> | <img width="800" alt="Home Page" src="https://github.com/balatstar/budgetbuddy/assets/12745474/10767b6a-9609-4e68-b7d2-74e5950e99ae">

**Expected:**

![image](https://user-images.githubusercontent.com/12745474/227751375-40282971-5bde-49ca-9612-8cc1be58ba99.png)